src/content/effective-dart/design.md

```diff
@@ -652,11 +653,13 @@ implicit interfaces except for classes that are clearly intended to be
 implemented. Otherwise, you may introduce a coupling that the author doesn't
 intend, and they may break your code without realizing it.
 
-### DO document if your class supports being used as an interface
-
-If your class can be used as an interface, mention that in the class's doc
-comment.
+### DO use class modifiers to control if your class can be an interface
 
+When designing a library, use class modifiers like `final`, `base`, or `interface` to enforce intended
```

Comment:
When it comes to `implements` clauses, `interface` has no effect, but `sealed` does:
```suggestion
When designing a library, use class modifiers like `final`, `base`, or `sealed` to enforce intended
```